pkg/core/storage/memcached_store.go

```diff
@@ -155,9 +161,14 @@ func (s *MemCachedStore) seek(ctx context.Context, rng SeekRange, cutPrefix bool
 	}
 	ps := s.ps
 	s.mut.RUnlock()
+
+	less := func(k1, k2 []byte) bool { return bytes.Compare(k1, k2) < 0 }
```

Comment:
Maybe inline this (if you also find it readable)?
```
res := bytes.Compare(k1, k2)
return res != 0 && rng.Backwards == (res > 0)
```

Reply:
Yes, nice! Fixed.